Miniforge3/construct.yaml

```diff
@@ -28,11 +30,9 @@ specs:
 {% else %}
   - python 3.10.*
 {% endif %}
-
-{% if name.startswith("Mambaforge") %}
-  - mamba 1.4.2
-{% endif %}
   - conda {{ version.split("-")[0] }}
+  - conda-libmamba-solver {{ conda_libmamba_solver_version }}
+  - mamba {{ mamba_version }}
```

Comment:
Ah nvm looks like the version is tied to how Conda & friends are versioned. Submitted PR ( https://github.com/conda-forge/miniforge/pull/480 ) to address this